Resources/Prototypes/Entities/Constructible/Ground/pipes.yml

```diff
@@ -20,26 +22,143 @@
 
 - type: entity
   parent: PipeBase
-  id: FourwayPipe
-  name: Fourway Pipe
+  id: PipeLongitudinal
+  suffix: Longitudinal
   components:
   - type: Icon
-    texture: Constructible/Atmos/pipe.rsi/pipeFourway2.png
+    texture: Constructible/Atmos/pipe.rsi/pipeLongitudinal2.png
   - type: NodeContainer
     nodes:
     - !type:PipeNode
       nodeGroupID: Pipe
-      pipeDirection: Fourway
+      pipeDirection: Longitudinal
 
 - type: entity
   parent: PipeBase
-  id: LongitudinalPipe
-  name: Longitudinal Pipe
+  id: PipeLateral
+  suffix: Lateral
   components:
   - type: Icon
-    texture: Constructible/Atmos/pipe.rsi/pipeLongitudinal2.png
+    texture: Constructible/Atmos/pipe.rsi/pipeLateral2.png
   - type: NodeContainer
     nodes:
     - !type:PipeNode
       nodeGroupID: Pipe
-      pipeDirection: Longitudinal
+      pipeDirection: Lateral
+
+- type: entity
+  parent: PipeBase
+  id: PipeNWBend
+  suffix: NWBend
+  components:
+  - type: Icon
+    texture: Constructible/Atmos/pipe.rsi/pipeNWBend2.png
+  - type: NodeContainer
+    nodes:
+    - !type:PipeNode
+      nodeGroupID: Pipe
+      pipeDirection: NWBend
+
+- type: entity
+  parent: PipeBase
+  id: PipeNEBend
+  suffix: NEBend
+  components:
+  - type: Icon
+    texture: Constructible/Atmos/pipe.rsi/pipeNEBend2.png
+  - type: NodeContainer
+    nodes:
+    - !type:PipeNode
+      nodeGroupID: Pipe
+      pipeDirection: NEBend
+
+- type: entity
+  parent: PipeBase
+  id: PipeSWBend
+  suffix: SWBend
+  components:
+  - type: Icon
+    texture: Constructible/Atmos/pipe.rsi/pipeSWBend2.png
+  - type: NodeContainer
+    nodes:
+    - !type:PipeNode
+      nodeGroupID: Pipe
+      pipeDirection: SWBend
+
+- type: entity
+  parent: PipeBase
+  id: PipeSEBend
+  suffix: SEBend
+  components:
+  - type: Icon
+    texture: Constructible/Atmos/pipe.rsi/pipeSEBend2.png
+  - type: NodeContainer
+    nodes:
+    - !type:PipeNode
+      nodeGroupID: Pipe
+      pipeDirection: SEBend
```

Comment:
I wonder if it'd be best to just have a single bent pipe and simply change rotation with `RotatableComponent` or something similar.